src/components/DeletedButton/index.jsx

```diff
@@ -0,0 +1,9 @@
+function DeletedButton (){
```

Comment:
See my comment on your Dropdown component - this DeletedButton should be replaced with a single re-usable component that can be used for all nav links